### PR TITLE
[Feat] - add bank class and create methods for deposit and withdraw

### PR DESCRIPTION
**Pull request by abosiy**

### CODE REVIEW

File: src/com/bosiy/BankAccount.java [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        if (value > this.balance) {
+            System.out.println("Your balance is low!");
+            return;
+        }
+        this.balance -= value;
+        System.out.println("Your balance: " + this.balance);
+    }
+
+    public void transferTo(double value, BankAccount receiver) {
+        if (this.balance >= value - 3.5) {
```
Comment (scratchw0w) [inline src/com/bosiy/BankAccount.java]: Move 3.5 to const, name it

File: src/com/bosiy/BankAccount.java [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            this.balance = this.balance - value - 3.5;
```
Comment (scratchw0w) [inline src/com/bosiy/BankAccount.java]: Value - 3.5 is repeated twice, so we can move it to local var

**Review by scratchw0w — CHANGES_REQUESTED** [review]

File: src/com/bosiy/BankAccount.java (hunk 1 above)
Comment (abosiy) [reply src/com/bosiy/BankAccount.java]: fixed

File: src/com/bosiy/BankAccount.java (hunk 2 above)
Comment (abosiy) [reply src/com/bosiy/BankAccount.java]: fixed

File: src/com/bosiy/BankAccount.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        }
+        this.balance -= value;
+        System.out.println("Your balance: " + this.balance);
+    }
+
+    public void transferTo(double value, BankAccount receiver) {
+        double transactionCost = value - COMMISSION;
+
+        if (this.balance >= transactionCost) {
+            this.balance = this.balance - transactionCost;
```
Comment (scratchw0w) [inline src/com/bosiy/BankAccount.java]: May be simplified

File: src/com/bosiy/BankAccount.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    public double getBalance() {
+        return this.balance;
+    }
+
+    public void setBalance(double balance) {
+        this.balance = balance;
+    }
+
+    public void deposit(double value) {
+        if (value <= 0) {
```
Comment (scratchw0w) [inline src/com/bosiy/BankAccount.java]: Can be moved to separate method cause used in 2 different methods

**Review by scratchw0w — CHANGES_REQUESTED** [review]

File: src/com/bosiy/BankAccount.java (hunk 3 above)
Comment (abosiy) [reply src/com/bosiy/BankAccount.java]: fixed

File: src/com/bosiy/BankAccount.java (hunk 4 above)
Comment (abosiy) [reply src/com/bosiy/BankAccount.java]: fixed

File: src/com/bosiy/BankAccount.java [hunk 5]
```diff
@@ -37,17 +37,22 @@ public void setBalance(double balance) {
         this.balance = balance;
     }
 
+    public boolean checkOut(double value) {
```
Comment (scratchw0w) [inline src/com/bosiy/BankAccount.java]: rename it, to: isNegative

**Review by scratchw0w — CHANGES_REQUESTED** [review]

File: src/com/bosiy/BankAccount.java (hunk 5 above)
Comment (abosiy) [reply src/com/bosiy/BankAccount.java]: fixed

**Comment by scratchw0w** [comment]
Great Job!

**Review by scratchw0w — APPROVED** [review]

**Pull request merged**